src/main/java/org/spongepowered/asm/mixin/transformer/MixinProcessor.java

```diff
@@ -269,9 +271,9 @@ public void audit(MixinEnvironment environment) {
         }
     }
 
-    synchronized boolean applyMixins(MixinEnvironment environment, String name, ClassNode targetClassNode) {
+    synchronized ClassNode applyMixins(MixinEnvironment environment, String name, Supplier<ClassNode> targetClassNode) {
```

Comment:
Don't think the return type needs to change, if the caller wants the node it can get it from the supplier.

Reply:
That's a fair idea. I was working forwards from `MixinTransformer#transformClass` where it seemed more convenient to returned the transformed `ClassNode`.

Reply:
This has been switched back to what it was.